### ISSUE TITLE
Broken Tool Tip

### ISSUE THREAD

**Issue by LizardMods** [imrayya/stable-diffusion-webui-Prompt_Generator#15]
It took some time trying to figure out why my tooltips were broken in [AUTOMATIC1111](https://github.com/AUTOMATIC1111)

[stable-diffusion-webui](https://github.com/AUTOMATIC1111/stable-diffusion-webui)

After exhausting all community posts to try and resolve the problem I unloaded all of my extensions and fixed my tool tips. I started loading extensions 1 by 1.

I discovered that this extension breaks the tooltips.

I don't like making bug reports so I hope this finds you in good spirits.

**Comment by LizardMods** [comment]
I have done some further testing. with the prompt generator installed, the tool tips work for the Prompt Generator tab only. My best guess is that the hints.js is overriding the 
stable-diffusion-webui\javascript\hints.js

**Comment by Harvester62** [comment]
I found four extensions in my local installation which use the hinting system, two of them being a potential source of this problem. The issue, anyway, seems not too complex to fix, perhaps just renaming the hints file and looking at the code content for the other two hints files.
https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/7534#issuecomment-1455946524

**Comment by imrayya** [comment]
Unfortunately, I am currently away from my workstation so I am unable to test if changing the file name as suggested by @Harvester62 will fix it for the next month or so. Since it affecting more than just this extension, if someone could make a pull request with the changes and test that it fixes the bug, that would be a great help.

**Comment by Harvester62** [comment]
Hi @imrayya I modified your hints.js both renaming the file to prompt_generator_hints.js and adapting the code inside it. I have just tested it and the problem is now fixed. I had also to do the same with the hints.js file of the extension/script gif2gif otherwise the overwriting of the hints.js inside ROOT\javascript would have not been fixed; for this other extension I will write to the related author. I am ready to do a PR but I think I would have to fork your repository and create a new branch to be able to do it, right?
